README.md

```diff
@@ -67,100 +67,100 @@ mv /tmp/INAV\ Configurator /opt/inav/inav-configurator
 2. If you have 3D model animation problems, enable "Override software rendering list" in Chrome flags chrome://flags/#ignore-gpu-blacklist
 
 ### Mac
 
 1. Visit [release page](https://github.com/iNavFlight/inav-configurator/releases)
-1. Download Configurator for Mac platform
+1. Download Configurator for the Mac platform
 1. Extract ZIP archive
 1. Run INAV Configurator
 
 ## Building and running INAV Configurator locally (for development)
 
-For local development, **node.js** build system is used.
+For local development, the **node.js** build system is used.
 
 1. Install node.js
-1. From project folder run `npm install`
+1. From the project folder run `npm install`
 1. To build the JS and CSS files and start the configurator:
     - With NW.js: Run `npm start`.
     - With Chrome: Run `npm run gulp`. Then open `chrome://extensions`, enable
-    the `Developer mode`, click on the `Load unpacked extension...` button and select the `inav-configurator` directory.
+    the `Developer mode`, click on the `Load unpacked extension...` button, and select the `inav-configurator` directory.
 
 Other tasks are also defined in `gulpfile.js`. To run a task, use `node ./node_modules/gulp/bin/gulp.js task-name`. Available ones are:
 
 - **build**: Generate JS and CSS output files used by the configurator from their sources. It must be run whenever changes are made to any `.js` or `.css` files in order to have those changes appear
 in the configurator. If new files are added, they must be included in `gulpfile.js`. See the comments at the top of `gulpfile.js` to learn how to do so. See also the `watch` task.
 - **watch**: Watch JS and CSS sources for changes and run the `build` task whenever they're edited.
-- **dist**: Create a distribution of the app (valid for packaging both as a Chrome app or a NW.js app)
+- **dist**: Create a distribution of the app (valid for packaging both as a Chrome app or NW.js app)
 in the `./dist/` directory.
 - **release**: Create NW.js apps for each supported platform (win32, osx64 and linux64) in the `./apps`
-directory. Running this task on macOS or Linux requires Wine, since it's needed to set the icon
-for the Windows app. If you don't have Wine installed you can create a release by running the **release-only-linux** task.
-<br>`--installer` argument can be added to build installers for particular OS. NOTE: MacOS Installer can be built with MacOS only.
+directory. Running this task on macOS or Linux requires Wine since it's needed to set the icon
+for the Windows app. If you don't have Wine installed you can create a release by running the **release-only-Linux** task.
+<br>`--installer` argument can be added to build installers for a particular OS. NOTE: MacOS Installer can be built with MacOS only.
 
 To build a specific release, use the command `release --platform="win64"` for example.
 
 ### Running with debug | Inspector
 
-To be able to open Inspector, you will need SDK flavours of NW.js
```

Comment:
Flavours is the correct spelling. Please revert this change.

Reply:
Sorry, no need for the uppercase F. That was only there as it was the start of a sentence.